todo_app/src/view.py

```diff
@@ -0,0 +1,18 @@
+from dataclasses import dataclass
+from typing import ClassVar, Literal
+
+from .trello import Item
+
+
+@dataclass
+class ViewModel:
+    items: list[Item]
+    done_status: ClassVar[Literal["Done"]] = "Done"
+
+    @property
+    def done_items(self):
+        return filter(lambda item: item.status == self.done_status, self.items)
```

Comment:
Not important here, but just in case not a conscious decision - worth noting that this returns you an iterator rather than a list, so you _may_ want to wrap it in a list:
```suggestion
        return list(filter(lambda item: item.status == self.done_status, self.items))
```

That will be necessary if we want to count the items (e.g. in the test mentioned above). You can [read a bit more about that here](https://stackoverflow.com/questions/25653996/what-is-the-difference-between-list-and-iterator-in-python) if curious